src/core/hll.c

```diff
@@ -107,6 +107,9 @@ void set_max_inline_size(MVMThreadContext *tc, MVMObject *config_hash, MVMHLLCon
             config->max_inline_size = MVM_repr_get_int(tc, size);
     });
 }
+void MVM_hll_set_config_key(MVMThreadContext *tc, MVMHLLConfig *hll, MVMString *key, MVMObject *value) {
+    hll->uint_box_type = value;
+}
```

Comment:
Shouldn't this use `key` somehow?

Reply:
If you've ever needed any proof of my laziness, this is is as good as any :D
Yes, it should. Just wasn't required to make progress as the syscall is currently only ever used for setting uint_box_type (and not even that yet in this branch)